chcemvediet/static/main/css/navs.sass

```diff
@@ -24,7 +21,9 @@
       &:focus,
         border: none
         color: $chv-color-blue
+  > li
+    + li
+      @include chv-baseline($chv-fontsize-li, 1, 0, 0.5)
+      margin-bottom: -1px
   li:not(:last-child)
     border-bottom: 1px solid $nav-tabs-border-color
-  @media (max-width: $screen-xs-max)
-    margin-top: 35px
```

Comment:
Aky ma vyznam nahradit implicitny margin v cssku, explicitnym spacerom v html? Nie je rozumnejsie mat tuto medzeru priamo v komponente? Len samozrejme namiesto `35px` treba pouzit vhodny nasobok baseline.

In general sby som sa explicitnym spacer elementom v html vyhybal, kde sa to da spravit nejak implicitne.

Reply:
Podobne ako je v `typo.sass` implicitny padding v css.
```
li, .chv-li
  &:last-child
    padding-bottom: ($chv-baseline - chv_baseline_fix($chv-fontsize-li, 1))
    ...
```
, kvoli tomu sme v `chv-nav` museli prepisovat last-child. Ak opätovne niekde v kode by sme chceli pouzit `chv-nav` bude to aj  s `margin-top` pri `xs` obrazvke, co moze byt nechcenne. :ok: prerobil som to.

Reply:
> Ak opätovne niekde v kode by sme chceli pouzit `chv-nav` bude to aj  s `margin-top` pri `xs` obrazvke, co moze byt nechcenne

To budeme riesit potom.